src/lib/bind/effects.html

```diff
@@ -14,10 +14,15 @@
 
     _shouldAddListener: function(effect) {
       return effect.name &&
-             effect.mode === '{' &&
-             !effect.negate &&
-             effect.kind != 'attribute'
-             ;
+             effect.kind != 'attribute' &&
+             effect.kind != 'text' &&
+             !effect.isCompound &&
+             effect.parts[0].mode === '{' &&
+             !effect.parts[0].negate;
+    },
+
+    _compoundInitializationEffect: function(source, value, effect) {
```

Comment:
Instead of doing this, let's make the annotator just leave the literal parts of the binding in the node.